simphony_ui/liggghts_model/liggghts_model.py

```diff
@@ -120,64 +122,68 @@ class LiggghtsModel(HasStrictTraits):
                     show_border=True
                 ),
                 VGroup(
                     Item(
                         name='wall_young_modulus',
-                        label='Young\'s modulus'
+                        label='Young\'s modulus (Pa)'
                     ),
                     Item(
                         name='wall_poisson_ratio',
                         label='Poisson\'s ratio'
                     ),
                     VGroup(
                         Item(
-                            name='wall_restitution_coefficient_wall',
-                            label='with wall particles'
-                        ),
-                        Item(
                             name='wall_restitution_coefficient_flow',
                             label='with flow particles'
                         ),
-                        label='Restitution Coefficient'
-                    ),
-                    VGroup(
                         Item(
-                            name='wall_friction_coefficient_wall',
+                            name='wall_restitution_coefficient_wall',
                             label='with wall particles'
                         ),
+                        label='Restitution Coefficient'
+                    ),
+                    VGroup(
                         Item(
                             name='wall_friction_coefficient_flow',
                             label='with flow particles'
                         ),
-                        label='Friction Coefficient'
-                    ),
-                    VGroup(
                         Item(
-                            name='wall_cohesion_energy_density_wall',
+                            name='wall_friction_coefficient_wall',
                             label='with wall particles'
                         ),
+                        label='Friction Coefficient'
+                    ),
+                    VGroup(
                         Item(
                             name='wall_cohesion_energy_density_flow',
                             label='with flow particles'
                         ),
-                        label='Cohesion energy density'
+                        Item(
+                            name='wall_cohesion_energy_density_wall',
+                            label='with wall particles'
+                        ),
+                        label='Cohesion energy density (J/m^3)'
                     ),
                     Item(name='wall_pair_potentials'),
-                    label='Wall particles parameters',
+                    label='Wall Particles Parameters',
                     show_border=True
                 ),
-                label='System parameters/ conditions',
+                label='System Parameters / Conditions',
                 show_border=True
             )
         )
     )
 
+    @on_trait_change('input_file')
+    def is_valid(self):
```

Comment:
use `update_valid`. It sounds too much like a property this way. Just as a reminder, never use e.g. `_valid_changed` because it would connect automatically (no need for `on_trait_change`) but it's less flexible and there are potential collisions between the two methods.